platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -352,17 +429,22 @@ class HangingProtocolService {
       // DisplaySets for the viewport, Note: this is not the actual displaySet,
       // but it is a info to locate the displaySet from the displaySetService
       let displaySetsInfo = [];
-      viewport.displaySets.forEach(({ id, options: displaySetOptions }) => {
-        const viewportDisplaySet = this.displaySetMatchDetails.get(id);
+      viewport.displaySets.forEach(({ id, displaySetIndex = 0, options: displaySetOptions }) => {
```

Comment:
can you please document why `displaySetIndex` is destructured here? I don't recall havving that in the HP